### DISCUSSION TITLE
Week 6

### DISCUSSION THREAD

**Discussion by gabebott** [USU-ITLS-x265-23-summer/class-repo#134] [General]
I really enjoyed the video and the article this week. I think that both showed a different side of accessibility. The video did a great job of bringing up the point that we live in a world that is catered toward able-bodied people. This can cause issues and challenges for those that have disabilities. Because of this there is an ever-present need for evolution towards a more inclusive and accessible world for everybody.

In the article it went over the story of Mihai. I think that Mihai's story was very touching and it showed a different angle on accessibility. I think an aspect of accessibility that isn't often touched on is the ability to access a community. By being able to contribute and be part of a team or a collective, this is accessibility. In Mihai's case, he was struggling with a debilitating illness that left him struggling with everyday tasks. He was able to find a way to be part of a team, help people, and make contributions. This was a mental accessibility as much as a physical accessibility for him.

In the freecodecamp activities I found that they were a bit easier to understand this week. In the past few weeks I have struggled to really grasp the complete concept, but I found these ones easier to digest. I enjoyed the quiz building one, as that is an idea for a project that I had.

The two project ideas that I have narrowed down to are a personal website/ UX portfolio, and a fun memory game. I like the idea of having a project that I can use in my personal life to display my portfolio, but I also like the idea of making something fun that I can share with people. Some challenges that I anticipate with building a personal website are:

- Design and Layout
- Content Organization
- Responsive Design
- Cross-Browser Compatibility
- Hosting and Deployment

Challenge that I anticipate when creating a memory game are:

- Game Logic and Rules
- Card Matching Logic
- Game State Management
- User Experience 
- Testing and Bug Fixing

**Comment by metasean** [comment]
> I think that Mihai's story was very touching and it showed a different angle on accessibility. I think an aspect of accessibility that isn't often touched on is the ability to access a community. By being able to contribute and be part of a team or a collective, this is accessibility. In Mihai's case, he was struggling with a debilitating illness that left him struggling with everyday tasks. He was able to find a way to be part of a team, help people, and make contributions. This was a mental accessibility as much as a physical accessibility for him.

💯%!

> In the freecodecamp activities I found that they were a bit easier to understand this week.

🙌

**Comment by Skye404** [comment]
I like your idea of the memory game, so I kind hope you go with that project, so I can try it out!

I agree that a part of accessibility that is often forgotten is being able to "access a community," as you say. When we create accessible websites, we are allowing a group of people to enter the discussion/community. Without accessibility, we unintentionally (or intentionally because we are aware) exclude people from our work.
